extensions/blocks/podcast-player/style.scss

```diff
@@ -243,10 +243,8 @@ $player-background: transparent;
 		fill: var(--jetpack-podcast-player-primary);
 
 		svg {
-			display: inline-block;
-			vertical-align: middle;
+			position: absolute;
```

Comment:
I was hoping we could figure out why the `inline-block` adds an extra pixel in the editor 🤓. I also liked the fact that the inline style makes it centered regardless of the title `font-size`, while the absolute position requires specifying offset (not in this font-size/icon-size combination, though!).

Anyways, while knowing what caused the 1px push would make me happy I think it's a minor and I don't think we should spend more time on this if the absolute position works. I'd just make sure that it's fine in other browsers. I remember e.g. Safari sometimes reacting differently to the absolute position!

Reply:
I hadn't considered lots of those. Thanks for the comment! 

I checked into it in the editor view more and it doesn't seem like the `inline-block` or `vertical-align` properties _do anything_. I increased the font-size of the track list to make it more obvious.

![editor-vertical-align](https://user-images.githubusercontent.com/967608/78696333-e5a5e880-78c4-11ea-9dcf-ed0ed70ef939.gif)

I think the thing that's pushing the icon down to line up more or less with the center of the line is the `line-height`.

Reply:
The extra height on the wrapping `span` is due to the same thing as `images` having extra height on their wrapping elements: https://codepen.io/beau/pen/EjBovR

Reply:
I looked at this for way too long 😅 FWIW - the `position: absolute;` looks good in Safari, FF, and Chrome.

Reply:
Thanks for looking into it and all the details! <3 Absolute position looks good 👍